tests/integration/projects/general/tests/test_meta.py

```diff
@@ -68,3 +69,15 @@ def has_cors_headers(headers):
         assert_status=200,
         assert_headers=has_cors_headers,
     )
+
+
+@pytest.mark.parametrize(
+    "metrics",
+    ['_mb_request_duration_seconds_count', '_request_duration_seconds_bucket'],
```

Comment:
these metrics will only show up after receiving the first inference request right? That seems to be the behavior I observed on Mac OS. Could that be the reason the API integration tests fail on Mac OS? perhaps we can send a prediction request before checking the "/metrics" endpoint in this test?

Reply:
the ``_mb`` metrics only shows up if the server is enable microbatching.

Reply:
@aarnphm https://github.community/t/why-is-docker-not-installed-on-macos/17017/7